src/pyfme/aircrafts/cessna_310.py

```diff
@@ -349,11 +331,69 @@ def get_engine_force(delta_t):
 
         q_cruise = 91.2 * 47.880172   # Pa
 
-        Sw = geometric_Data()[0]
+        Sw = geometric_data()[0]
 
         engine_force = Ct * Sw * q_cruise   # N
 
     else:
         raise ValueError('delta_t must be between 0 and 1')
 
-    return engine_force
\ No newline at end of file
+    return engine_force
+
+
+def get_forces_and_moments(TAS, rho, alpha, beta, delta_e, ih, delta_ail,
+                           delta_r, delta_t, attitude):
+    """Return the total forces and moments including aerodynamics, thrust and
+    gravity.
+
+    Parameters
+    ----------
+    TAS : float
+        velocity (m/s).
+    rho : float
+        density (kg/m3).
+    alpha : float
+        attack angle (rad).
+    beta : float
+        sideslip angle (rad).
+    delta_e : float
+        elevator deflection (rad).
+    ih : float
+        stabilator deflection (rad).
+    delta_ail : float
+        aileron deflection (rad).
+    delta_r : float
+        rudder deflection (rad).
+    delta_t : float
+        Thrust level (between 0-1).
+    attitude : array_like
+        Attitude angles: (theta, phi, psi).
+
+    Returns
+    -------
+    forces : array_like
+        3 dimensional vector with (F_x_s, F_y_s, F_z_s) forces  in body axes.
+        (N)
+    moments : array_like
+         3 dimensional vector with (Mxs, Mys, Mzs) forces in body axes. (N·m)
+    """
+    # As stability axes are coincident with wind axes at the moment of
+    # linearization (with alpha=0 and beta=0), stability axes are parallel to
+    # body axes.
+    aero_forces = get_aerodynamic_forces(TAS, rho, alpha, beta, delta_e, 0,
+                                         delta_ail, delta_r)
+    thrust = get_engine_force(delta_t)
+    # Assuming that all the engine thrust is prodecued in the x_body direction
+    engine_force = np.array([thrust, 0, 0])
+
+    g0 = 9.81  # m/s²

```

Comment:
What about this hardcoded g0 value here? Is there a way to move it somewhere else?